setup.php

```diff
@@ -29,7 +29,7 @@
  * -------------------------------------------------------------------------
  */
 
-define("PLUGIN_SCCM_VERSION", "2.3.0");
+define("PLUGIN_SCCM_VERSION", "2.3.1");
```

Comment:
Since it will rely on another major core version
```suggestion
define("PLUGIN_SCCM_VERSION", "2.4.0");
```

Reply:
Will be done during release process.

Reply:
sorry, is for update test